src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsworkload/service/reduction/GetReductionService.kt

```diff
@@ -36,4 +37,8 @@ class GetReductionService(private val reductionsRepository: ReductionsRepository
         .fold(BigDecimal.ZERO) { first, second -> first.add(second) }
     } ?: BigDecimal.ZERO
     ).stripTrailingZeros()
+
+  fun findOutOfDateReductions(): List<ReductionEntity> = reductionsRepository.findByEffectiveToLessThanAndStatus(
```

Comment:
Do we want to deal with these at the same time? I might misunderstand but don't we want to set the ACTIVE ones to ARCHIVED and the SCHEDULED ones to ACTIVE?